src/pipecat/transports/services/livekit.py

```diff
@@ -605,6 +605,11 @@ async def _audio_in_task_handler(self):
                 pipecat_audio_frame = await self._convert_livekit_audio_to_pipecat(
                     audio_frame_event
                 )
+
+                # Skip frames with no audio data
+                if len(pipecat_audio_frame.audio) == 0:
+                    continue
```

Comment:
Mmmm, this is weird. Does this mean the resampler is returning 0?

Reply:
When I tried adding the streaming resampler the first time I believe I was seeing the resampler not always returning data, that's why I didn't use it. But I assumed I was doing something wrong. @filipi87 since you implemented the new resampler, you might now.